app/models/donation.rb

```diff
@@ -9,4 +9,6 @@ class Donation < ActiveRecord::Base
   validates :amount, presence: true, numericality: { greater_than: 0 }
 
   scope :search, -> (keyword) { joins(:donor).where("name ILIKE ?", "%#{keyword}%") }
+
+  scope :donations_per_year, -> (year) { where("EXTRACT(YEAR FROM created_at) = ?", year).sum(:amount).round }
```

Comment:
Scopes must guarantee to return a relation, so `.sum(:amount).round` needs to live outside of the scope. 😀 Other than that, I'm ready to merge this PR.